common/route.c

```diff
@@ -25,7 +25,7 @@ bool route_can_carry(const struct gossmap *map,
 		       struct amount_msat amount,
 		       void *arg)
 {
-	if (!c->half[dir].enabled)
+	if (!c->half[0].enabled || !c->half[1].enabled)
```

Comment:
This may touch uninitialized memory, since we may only have gotten one direction's update.

Reply:
yes, i know, its still draft ;)